doc/planning_scene_monitor/planning_scene_monitor_tutorial.rst

```diff
@@ -0,0 +1,47 @@
+
+PlanningScene
+-------------
+The :planning_scene:`PlanningScene` can be seen as a snapshot of the world and that includes both the RobotState and any number of collision objects. The Planning Scene can be used for collision checking as well as getting information about the environment.
+
+PlanningSceneMonitor
+--------------------
+The :planning_scene_monitor:`PlanningSceneMonitor` wraps a PlanningScene with ROS interfaces for keeping the PlanningScene up to date. To access the PlanningSceneMonitor's underlying PlanningScene, you can use the provided :planning_scene_monitor:`LockedPlanningSceneRW` and :planning_scene_monitor:`LockedPlanningSceneRO` classes.
+
+The PlanningSceneMonitor has the following objects which have their own ROS interfaces for keeping sub-components of the planning scene up to date:
+
+ * A :planning_scene_monitor:`CurrentStateMonitor` for tracking updates to the RobotState via a ``robot_state_subscriber_`` and a ``tf_buffer_``, as well as a planning scene subscriber for listening to planning scene diffs from other publishers.
+ * An OccupancyMapMonitor for tracking updates an OccupancyMap via ROS topics and services.
+
+The PlanningSceneMonitor has the following subscribers:
+
+ * ``collision_object_subscriber_`` - Listens to a provided topic for :moveit_msgs:`CollisionObject` messages that might add, remove or modify collision objects in the planning scene and passes them into its monitored planning scene
+ * ``planning_scene_world_subscriber_`` - Listens to a provided topic for :moveit_msgs:`PlanningSceneWorld` messages that may contain collision object information and or octomap information. This is useful for keeping planning scene monitors in sync
+ * ``attached_collision_object_subscriber_`` - Listens on a provided topic for :moveit_msgs:`AttachedCollisionObject` messages that determine the attaching / detaching of objects to links in the robot state.
+
+The PlanningSceneMonitor has the following services:
+
+ * ``get_scene_service_`` - Which is an optional service to get the full planning scene state.
+
+The PlanningSceneMonitor is initialized with:
+
+ * ``startSceneMonitor`` - Which starts the ``planning_scene_subscriber_``
+ * ``startWorldGeometryMonitor`` - Which starts the ``collision_object_subscriber_``, the ``planning_scene_world_subscriber_``, and the OccupancyMapMonitor.
+ * ``startStateMonitor`` - Which starts the CurrentStateMonitor and the ``attached_collision_object_subscriber_``.
+ * ``startPublishingPlanningScene`` - Which starts another thread for publishing the entire planning scene on a provided topic for other PlanningSceneMonitor's to subscribe to
+ * ``providePlanningSceneService`` - Which starts the ``get_scene_service_``.
+
+PlanningSceneInterface
+----------------------
+The The :planning_interface:`PlanningSceneInterface` is used for publishing updates to a MoveGroup's :planning_scene_monitor:`PlanningSceneMonitor`. It may not work without MoveGroup or MoveItCpp
```

Comment:
```suggestion
The :planning_interface:`PlanningSceneInterface` is a useful class for publishing updates to a MoveGroup's :planning_scene_monitor:`PlanningSceneMonitor` through a C++ API without creating your own subscribers and service clients. It may not work without MoveGroup or MoveItCpp
```